pkg/regopolicyinterpreter/regopolicyinterpreter.go

```diff
@@ -0,0 +1,557 @@
+package regopolicyinterpreter
```

Comment:
does it need to be exported as an hcsshim package from the start? Or can we for now put it under `/internal` and export later when needed?

Reply:
Arguments can be made for either. Ultimately I'll follow the project's preferences here, but I think if security policy is public this should be too.

Reply:
Moved.